confluent-kafka-plugins/pom.xml

```diff
@@ -255,8 +280,8 @@
         <version>1.1.0</version>
         <configuration>
           <cdapArtifacts>
-            <parent>system:cdap-data-pipeline[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-            <parent>system:cdap-data-streams[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
+            <parent>system:cdap-data-pipeline[6.8.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
```

Comment:
Make this 6.8.0 . The release is already complete.

Reply:
Replaced `6.8.0-SNAPSHOT` with `6.8.0` 2698de880d1d31e02abdfb05d2c3868a321c46c0